### ISSUE TITLE
"conda env config vars set" can't deal with "=" inside the value

### ISSUE THREAD

**Issue by mil-ad** [conda/conda#10603]
## Current Behavior

The `conda env config vars set` can't deal with environment variables that include `=`. For example:

```
$ conda env config vars set XLA_FLAGS=--xla_gpu_cuda_data_dir=/opt/cuda10
$ conda env config vars list
XLA_FLAGS = /opt/cuda10
```

I tried wrapping the whole value inside single or double quotations but it doesn't help 

## Expected Behavior

```
$ conda env config vars list
XLA_FLAGS = --xla_gpu_cuda_data_dir=/opt/cuda10
```

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

```

     active environment : coin
    active env location : /home/milad/miniconda3/envs/coin
            shell level : 2
       user config file : /home/milad/.condarc
 populated config files : /home/milad/.condarc
          conda version : 4.9.2
    conda-build version : not installed
         python version : 3.7.6.final.0
       virtual packages : __cuda=11.2=0
                          __glibc=2.33=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /home/milad/miniconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/milad/miniconda3/pkgs
                          /home/milad/.conda/pkgs
       envs directories : /home/milad/miniconda3/envs
                          /home/milad/.conda/envs
               platform : linux-64
             user-agent : conda/4.9.2 requests/2.24.0 CPython/3.7.6 Linux/5.4.108-1-MANJARO manjaro/21.0.1 glibc/2.33
                UID:GID : 1000:1000
             netrc file : /home/milad/.netrc
           offline mode : False
```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
==> /home/milad/.condarc <==
auto_activate_base: False
changeps1: False
ssl_verify: True
channels:
  - defaults

```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```

```
</p></details>

**Comment by nguyenducsang** [comment]
Ok

**Comment by jezdez** [comment]
This was fixed in conda 4.11.0 (#10905).
